plugins/cassandra/cassandraembed.cpp

```diff
@@ -3695,21 +3796,127 @@ class CassSortableIterator : public CassandraIterator
     bool descending;
 };
 
-class CassMultiIterator : public CInterface, implements IRowProvider, implements ICompare, implements IConstWorkUnitIterator
+interface IConstWorkUnitIteratorEx : public IConstWorkUnitIterator
```

Comment:
I prefer public (I think the macros we put in to supposedly make it easier to port to Java unfortunately serve to raise the entry barrier to new developers, and we should consider discontinuing their use)